SMLHelper/Utility/ExtBannedIdManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace SMLHelper.V2.Utility
+{
+    using System.IO;
+    using System.Collections.Generic;
+    using System;
+
+    internal static class ExtBannedIdManager
+    {
+        private static bool IsInitialized = false;
+
+        private static readonly Dictionary<string, List<int>> BannedIdDictionary = new Dictionary<string, List<int>>();
+
+        private const string BannedIdDirectory = @"./QMods/Modding Helper/RestrictedIDs";
```

Comment:
Maybe we could come up with a better name

Reply:
I'm always open to suggestions for better names.
Since this is a folder the end-user may be expected to interact with, it should be easily understood what it's purpose is.

Reply:
Any final word on this? I can go either way with this.